server/etcdserver/cluster_util.go

```diff
@@ -237,6 +237,7 @@ func isCompatibleWithVers(lg *zap.Logger, vers map[string]*version.Versions, loc
 func getVersion(lg *zap.Logger, m *membership.Member, rt http.RoundTripper) (*version.Versions, error) {
 	cc := &http.Client{
 		Transport: rt,
+		Timeout:   10 * time.Second,
```

Comment:
I think we should use ServerConfig.RequestTimeout instead of hardcoding a value.

Reply:
> I think we should use ServerConfig.RequestTimeout instead of hardcoding a value.

Now, it's has many clients that hardcode timeout.  
The struct ServerConfig not containe RequestTimeout,  Do we need to add ReqeustTimeout to ServerConfig ?

Reply:
FYI. [config.go#L316](https://github.com/etcd-io/etcd/blob/fb559105006337157635673f216c2f1392050f83/server/config/config.go#L316)

Reply:
> FYI. [(c *ServerConfig) ReqTimeout()](https://github.com/etcd-io/etcd/blob/fb559105006337157635673f216c2f1392050f83/server/config/config.go#L316)

Thanks! I will  use ServerConfig.RequestTimeout.